app/pages/_app.tsx

```diff
@@ -64,6 +65,34 @@ const formStyle = {
 export default function App({ Component, pageProps }) {
   return (
     <>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
Is Next throwing any warnings about putting `<meta>` here?

Reply:
Next was throwing errors having metadata in _document.tsx and recomended it to be put in _app.tsx

Reply:
perfect